test/Index/index_module_refs.swift

```diff
@@ -0,0 +1,29 @@
+// RUN: %target-swift-ide-test -print-indexed-symbols -enable-source-import -source-filename %s -I %S/Store/Inputs | %FileCheck %s
+
```

Comment:
It's shaping up great! One final (I think) suggestion about the test case; add a line like this:
```
import NonExistingModuleName // make sure there's no problem with invalid imports
```
as per comment this would be to guard that there's no issue occurring with invalid imports, like some crash/assertion hit while trying to index its module reference.

Reply:
Makes sense! I added `// CHECK-NOT: {{.*}} | NonExistingModuleName` to it to guarantee it doesn't even show up, but if it has performance implications we can leave it out.